plugins/woocommerce/includes/admin/class-wc-admin-assets.php

```diff
@@ -135,21 +135,23 @@ public function admin_scripts() {
-					'i18n_ajax_error'           => _x( 'Loading failed', 'enhanced select', 'woocommerce' ),
-					'i18n_input_too_short_1'    => _x( 'Please enter 1 or more characters', 'enhanced select', 'woocommerce' ),
-					'i18n_input_too_short_n'    => _x( 'Please enter %qty% or more characters', 'enhanced select', 'woocommerce' ),
-					'i18n_input_too_long_1'     => _x( 'Please delete 1 character', 'enhanced select', 'woocommerce' ),
-					'i18n_input_too_long_n'     => _x( 'Please delete %qty% characters', 'enhanced select', 'woocommerce' ),
-					'i18n_selection_too_long_1' => _x( 'You can only select 1 item', 'enhanced select', 'woocommerce' ),
-					'i18n_selection_too_long_n' => _x( 'You can only select %qty% items', 'enhanced select', 'woocommerce' ),
-					'i18n_load_more'            => _x( 'Loading more results&hellip;', 'enhanced select', 'woocommerce' ),
-					'i18n_searching'            => _x( 'Searching&hellip;', 'enhanced select', 'woocommerce' ),
-					'ajax_url'                  => admin_url( 'admin-ajax.php' ),
-					'search_products_nonce'     => wp_create_nonce( 'search-products' ),
-					'search_customers_nonce'    => wp_create_nonce( 'search-customers' ),
-					'search_categories_nonce'   => wp_create_nonce( 'search-categories' ),
-					'search_pages_nonce'        => wp_create_nonce( 'search-pages' ),
+					'i18n_no_matches'                 => _x( 'No matches found', 'enhanced select', 'woocommerce' ),
+					'i18n_ajax_error'                 => _x( 'Loading failed', 'enhanced select', 'woocommerce' ),
+					'i18n_input_too_short_1'          => _x( 'Please enter 1 or more characters', 'enhanced select', 'woocommerce' ),
+					'i18n_input_too_short_n'          => _x( 'Please enter %qty% or more characters', 'enhanced select', 'woocommerce' ),
+					'i18n_input_too_long_1'           => _x( 'Please delete 1 character', 'enhanced select', 'woocommerce' ),
+					'i18n_input_too_long_n'           => _x( 'Please delete %qty% characters', 'enhanced select', 'woocommerce' ),
+					'i18n_selection_too_long_1'       => _x( 'You can only select 1 item', 'enhanced select', 'woocommerce' ),
+					'i18n_selection_too_long_n'       => _x( 'You can only select %qty% items', 'enhanced select', 'woocommerce' ),
+					'i18n_load_more'                  => _x( 'Loading more results&hellip;', 'enhanced select', 'woocommerce' ),
+					'i18n_searching'                  => _x( 'Searching&hellip;', 'enhanced select', 'woocommerce' ),
+					'ajax_url'                        => admin_url( 'admin-ajax.php' ),
+					'search_products_nonce'           => wp_create_nonce( 'search-products' ),
+					'search_customers_nonce'          => wp_create_nonce( 'search-customers' ),
+					'search_categories_nonce'         => wp_create_nonce( 'search-categories' ),
+					'search_taxonomy_terms_nonce'     => wp_create_nonce( 'search-taxonomy-terms' ),
+					'search_product_attributes_nonce' => wp_create_nonce( 'search-product-attributes' ),
```

Comment:
I only added the above two, but had to re-indent everything to re-align correctly.